### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by AshutoshM29**
I have added the "Homepage" for the E-com website, with Navigation and Footer component. Please give feedback and review :)

Preview Link: https://urban-tail.netlify.app/

Files to check:
[Footer](src/components/Footer/Footer.jsx)
[Navigation](src/components/Navigation/Navigation.jsx)
[Homepage](src/pages/Homepage/Homepage.jsx)
[CSS ](src/pages/Homepage/homepage.css)

Files to avoid: 
BIN files
[ ]public/index.html
[ ]src/App.css

https://user-images.githubusercontent.com/89376505/159932418-c87aad25-14b3-4556-b67e-f9c7144ef129.mp4

### CODE REVIEW

File: src/pages/Homepage/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    brandBayer,
+    brandRuffwear,
+  } from '../../assets/images'
+
+
+function Homepage() {
+    return <div>
+    <main>
+    <h1 class="hero-txt">Purr-fect for your paws,<br/>
+            Every Pet's Essential<br/>
```
Comment (Sahil25061999) [inline src/pages/Homepage/Homepage.jsx]: replace class with className

**Review by Sahil25061999 — APPROVED** [review]
looks good suggested minor change, replace the a tags with Link when you do routing.

File: src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            <a href="https://twitter.com/Astomi_29?s=08"> <i className="icons-social fab fa-twitter"></i></a></li>
+        <li className="list-item">
+            <a href="https://www.instagram.com/the.astomi/"> <i className="icons-social fab fa-instagram"></i></a></li>
+        <li className="list-item">
+            <a href="https://www.linkedin.com/mwlite/in/ashutosh-mishra-b50423206"><i
+                    className="icons-social fab fa-linkedin"></i></a></li>
+
+    </ul>
+    <div className="with-love">Made with ❤️ by : Ashutosh</div>
+</footer>)
```
Comment (Dev-AkshatGupta) [inline src/components/Footer/Footer.jsx:49]: Clean code I like it Though I will suggest you to also mention some comments to make developer experience better

**Review by Dev-AkshatGupta — APPROVED** [review]
I can't suggest you a single change.
So please to look at your code.
The best thing was I was able to understand the code that where things are going from to where which is the best part of your code Kudos to you man

File: src/components/Footer/Footer.jsx (hunk 2 above)
Comment (AshutoshM29) [reply src/components/Footer/Footer.jsx:49]: will keep that in mind :)

**Pull request merged**